src/pages/resultsView/enrichments/ProteinEnrichmentsTab.tsx

```diff
@@ -24,35 +24,41 @@ export default class ProteinEnrichmentsTab extends React.Component<IProteinEnric
         this.props.store._selectedEnrichmentProteinProfile = molecularProfile;
     }
 
+    private readonly enrichmentAnalysisGroups = remoteData({
+        await: () => [this.props.store.unalteredSampleKeys],
+        invoke: () => {
+            const groups = [{
+                name: "Altered group",
+                description: "samples that have alterations in the query gene(s).",
+                nameOfEnrichmentDirection: "Over-expressed",
+                count: this.props.store.alteredSampleKeys.result!.length,
+                color: this.uniqueColorGetter()
```

Comment:
I think the same problem will happen. You should recreate uniqueColorGetter on every invoke to avoid it